### PR TITLE
Hardend Signal desktop profile

### PR DESCRIPTION
**Pull request by onovy**

### CODE REVIEW

File: etc/profile-m-z/signal-desktop.profile [hunk 1]
```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
 netfilter
 nodvd
 nogroups
+noroot
```
Comment (rusty-snake) [inline etc/profile-m-z/signal-desktop.profile:35]: AFAIK noroot break electron/chromium based program in newer versions (see #2946).

File: etc/profile-m-z/signal-desktop.profile [hunk 2]
```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
...
 notv
 nou2f
+novideo
+protocol unix,inet,inet6
```
Comment (rusty-snake) [inline etc/profile-m-z/signal-desktop.profile:39]: Same as noroot.

File: etc/profile-m-z/signal-desktop.profile [hunk 3]
```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
 netfilter
 nodvd
 nogroups
+noroot
 notv
 nou2f
+novideo
```
Comment (rusty-snake) [inline etc/profile-m-z/signal-desktop.profile:38]: IIRC signal has a video-chat. IDK if it is also supported by the desktop-version.

File: etc/profile-m-z/signal-desktop.profile [hunk 4]
```diff
@@ -32,12 +32,16 @@ caps.keep sys_admin,sys_chroot
...
+protocol unix,inet,inet6
 shell none
 
 disable-mnt
 private-dev
+private-etc machine-id,resolv.conf,fonts
```
Comment (rusty-snake) [inline etc/profile-m-z/signal-desktop.profile:44]: IMHO also these should be added.
TLS/SSL: `ca-certificates,ssl,pki,crypto-policies,nsswitch.conf`
Others: `alternatives,ld.so.cache,ld.so.conf,ld.so.conf.d,ld.so.preload`

Also sort it alphabetical.

File: etc/profile-m-z/signal-desktop.profile (hunk 3 above)
Comment (onovy) [reply etc/profile-m-z/signal-desktop.profile:38]: desktop version don't have video-chat. Only Android/iOS version

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**